pkg/openfeature/api.go

```diff
@@ -0,0 +1,119 @@
+package openfeature
+
+import (
+	"errors"
+	"github.com/go-logr/logr"
+	"github.com/open-feature/go-sdk/pkg/openfeature/internal"
+	"sync"
```

Comment:
In other files (e.g., client.go in this pkg) the following format is used
```suggestion
	"errors"
	"sync"
	
	"github.com/go-logr/logr"
	"github.com/open-feature/go-sdk/pkg/openfeature/internal"
```

Reply:
hmm, seems we do not enforce linting on these, for example, `gofumpt`. I will manually fix this.

Reply:
Fixed with the latest commit